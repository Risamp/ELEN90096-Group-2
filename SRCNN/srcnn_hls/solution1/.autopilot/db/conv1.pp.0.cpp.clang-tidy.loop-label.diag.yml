---
MainSourceFile:  'C:/SPB_Data/ELEN90096-Group-2/SRCNN/srcnn_hls/solution1/.autopilot/db/conv1.pp.0.cpp'
Diagnostics:     
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_18_1:  '
    FileOffset:      18
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807000
        Length:          0
        ReplacementText: 'VITIS_LOOP_18_1: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_22_2:  '
    FileOffset:      22
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807039
        Length:          0
        ReplacementText: 'VITIS_LOOP_22_2: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_24_3:  '
    FileOffset:      24
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807085
        Length:          0
        ReplacementText: 'VITIS_LOOP_24_3: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_28_4:  '
    FileOffset:      28
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807134
        Length:          0
        ReplacementText: 'VITIS_LOOP_28_4: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_31_5:  '
    FileOffset:      31
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807198
        Length:          0
        ReplacementText: 'VITIS_LOOP_31_5: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_32_6:  '
    FileOffset:      32
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807244
        Length:          0
        ReplacementText: 'VITIS_LOOP_32_6: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_34_7:  '
    FileOffset:      34
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807292
        Length:          0
        ReplacementText: 'VITIS_LOOP_34_7: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_35_8:  '
    FileOffset:      35
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\SRCNN\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807337
        Length:          0
        ReplacementText: 'VITIS_LOOP_35_8: '
...
