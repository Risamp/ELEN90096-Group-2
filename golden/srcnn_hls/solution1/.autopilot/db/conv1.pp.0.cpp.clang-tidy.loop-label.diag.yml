---
MainSourceFile:  'C:/SPB_Data/ELEN90096-Group-2/golden/srcnn_hls/solution1/.autopilot/db/conv1.pp.0.cpp'
Diagnostics:     
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_18_1:  '
    FileOffset:      18
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807003
        Length:          0
        ReplacementText: 'VITIS_LOOP_18_1: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_22_2:  '
    FileOffset:      22
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807049
        Length:          0
        ReplacementText: 'VITIS_LOOP_22_2: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_23_3:  '
    FileOffset:      23
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807094
        Length:          0
        ReplacementText: 'VITIS_LOOP_23_3: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_26_4:  '
    FileOffset:      26
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807145
        Length:          0
        ReplacementText: 'VITIS_LOOP_26_4: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_27_5:  '
    FileOffset:      27
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807202
        Length:          0
        ReplacementText: 'VITIS_LOOP_27_5: '
  - DiagnosticName:  xilinx-label-all-loops
    Message:         'Added loop label VITIS_LOOP_34_6:  '
    FileOffset:      34
    FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\src\conv1.cpp'
    Replacements:    
      - FilePath:        'C:\SPB_Data\ELEN90096-Group-2\golden\srcnn_hls\solution1\.autopilot\db\conv1.pp.0.cpp'
        Offset:          807397
        Length:          0
        ReplacementText: 'VITIS_LOOP_34_6: '
...
